### ISSUE TITLE
Category Microsoft Edge ID 10952, 10953

### ISSUE THREAD

**Issue by FLeven** [scipag/HardeningKitty#28]
When executing, on Windows 11 22H2:
`Invoke-HardeningKitty -Mode Audit -FileFindingList .\lists\finding_list_msft_security_baseline_windows_11_22h2_machine.csv`
 
the two Edge settings are not found:
`[*] 1/29/2023 6:22:34 PM - Starting Category Microsoft Edge

[$] ID 10952, Configure Windows Defender SmartScreen, Result=, Recommended=1, Severity=Medium

[$] ID 10953, Prevent bypassing Microsoft Defender SmartScreen prompts for sites, Result=, Recommended=1, Severity=Medium`

the list points to the reg key:
**HKLM:\Software\Policies\Microsoft\MicrosoftEdge\PhishingFilter,EnabledV9**

I could only find this setting here (the second Defender setting is also found here):
**SOFTWARE\Policies\Microsoft\Internet Explorer\PhishingFilter**

**Comment by 0x6d69636b** [comment]
According to the document _MS Security Baseline Windows 11 v22H2.xlsx_  [URL](https://techcommunity.microsoft.com/t5/microsoft-security-baselines/windows-11-version-22h2-security-baseline/ba-p/3632520), the path _...Microsoft\MicrosoftEdge\PhishingFilter..._ is correct. However, the policy file _MSFT-Win11-v22H2.PolicyRules_ only points to the Internet Explorer one. I don't have a Windows 11 installation around at the moment, could you please configure the policy via group policy and monitor with registry key is changed?

**Comment by FLeven** [comment]
The settings are here:
Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\System

EnableSmartScreen

ShellSmartScreenLevel, which can be **Block** or Warn, **Block** would be the correct setting.

Don't get confused, the config about Explorer not Edge, I doubt this is any different from win10.

**Comment by 0x6d69636b** [comment]
Here a output of the Microsoft documentation regarding those settings, which use different path for Explorer and Edge

![image](https://user-images.githubusercontent.com/13913706/215543968-f82a8e6d-8b3e-4b55-bc00-314ec14fb30f.png)

These settings (rows 2482-2484) are covered in ID 10951 to 10954

**Comment by FLeven** [comment]
Nope,

**10952** and 10953 should match **2842 or 1043** (2 registry path entrys here) in your excel file.
**Computer\HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\System\**

Keys:
EnableSmartScreen
ShellSmartScreenLevel

but in your list it matches 2483 and 2484:
`10952,"Microsoft Edge","Configure Windows Defender SmartScreen",Registry,,**HKLM:\Software\Policies\Microsoft\MicrosoftEdge\PhishingFilter,EnabledV9**,,,,,1,=,Medium

10953,"Microsoft Edge","Prevent bypassing Microsoft Defender SmartScreen prompts for sites",Registry,,**HKLM:\Software\Policies\Microsoft\MicrosoftEdge\PhishingFilter,PreventOverride**,,,,,1,=,Medium
`

That's why the settings are reported as "unset", all other Defender settings are ok and don't fail. File Explorer Defender settings are in system and not in any *Edge registry folder.

**Comment by 0x6d69636b** [comment]
I don't get it:
-  ID 1043 matches ID's 10766 and 10767
-  ID 2482 matches ID's 10951 and 10954
-  ID 2483 matches ID 10952
-  ID 2484 matches ID 10953

Okay, the category _Microsoft Edge_ of 10952 and 10953 is not as in the Excel list however the registry keys are matching

**Comment by FLeven** [comment]
Ok, I missed the category before "Starting category Microsoft Edge", but they are still 10952 and 10953 empty and not used, even if add the Edge 107 Baseline, it is still: Result=

**Comment by 0x6d69636b** [comment]
That is interesting, I had access to a Win 11 Enterprise machine and used gpedit.msc and monitored for registry changes. Both registry keys were created after configuring the settings:

![image](https://user-images.githubusercontent.com/13913706/216068677-6d0ddfef-0cef-4b9e-82bf-7c48d5584ebf.png)

Is there an error in the source file from Microsoft? I did not check those templates yet.

**Comment by FLeven** [comment]
Enterprise feature ?

**Comment by 0x6d69636b** [comment]
Let's Microsoft answer this question, the documentation or the policy files are different. I posted a comment: https://techcommunity.microsoft.com/t5/microsoft-security-baselines/windows-11-version-22h2-security-baseline/bc-p/3733816/highlight/true#M562

If Microsoft recommends configuring Windows Defender for Mircosoft Edge, then the settings must be set in the way of the HardeningKitty list. If not this is not a recommendation I'll remove the checks.

I did a check on Windows 11 Pro as well, if I use gpedit.msc then the registry keys are set. If I use the PowerShell script of Microsoft then it is not configured.
